python/lsst/ts/scheduler/config_schema.py

```diff
@@ -116,440 +129,446 @@ class inside the module.
+                  during a slew. FALSE locks the rotator.
+                default: True
+              resume_angle:
+                type: boolean
+                description: >-
+                  If True enables the rotator to keep the image angle after a
+                  filter change, moving back the rotator to the previous angle
+                  after the rotator was placed in filter change position. If the
+                  flag is False, then the rotator is left in the filter change
+                  position. This must be TRUE to allow any movement of the
+                  rotator after a filter change.
+                default: true
+            additionalProperties: false
+          camera:
+            type: object
+            properties:
+              readout_time:
+                type: number
+                decription: Camera readout time (seconds).
+                default: 2.0
+              shutter_time:
+                type: number
+                description: Time it takes to open/close camera shutter (seconds).
+                default: 1.0
+              filter_change_time:
+                type: number
+                description: Time it takes to perform a filter change (seconds).
+                default: 120.0
+              filter_max_changes_burst_num:
+                type: number
+                description: Maximum number of filter changes in a night.
+                default: 1.
+              filter_max_changes_burst_time:
+                type: number
+                description: Minimum time (units=seconds) between filter changes in a night.
+                default: 0
+              filter_max_changes_avg_num:
+                type: number
+                description: Maximum average number of filter changes per year.
+                default: 3000
+              filter_max_changes_avg_time:
+                type: number
+                description: Maximum time (units=seconds) for the average number of filter changes.
+                default: 31557600.0
+              filter_mounted:
+                type: array
+                description: >-
+                  Initial state for the mounted filters. Empty positions must be
+                  filled with id="" no (filter).
+                default: [g, r, i, z, y]
+              filter_removable:
+                type: array
+                description: List of mounted filters that are removable for swapping
+                default: [y, z]
+              filter_unmounted:
+                type: array
+                description: List of unmounted but available filters to swap
+                default: [u]
+            additionalProperties: false
+          optics_loop_corr:
+            type: object
+            properties:
+              tel_optics_ol_slope:
+                type: number
+                description: >-
+                    Delay factor for Open Loop optics correction,  in units of seconds/(degrees in ALT slew)
```

Comment:
`,  in units of seconds/(degrees in ALT slew)` -> `(seconds/(degrees in ALT slew)).`. If you decide to leave as is then at least replace `  ` with ` ` and terminate the sentence in `.`.